episodes/08-motivation.md

```diff
@@ -146,7 +146,7 @@ A few ways to invite participation are:
 ### Encourage a Growth Mindset
 
 People vary in their beliefs about the nature of intelligence and skill development. In academic environments, people
-are often praised as "talented" or having "high ability," and may develop an identity around being a certain "type of
+are often praised as *talented* or having *high ability*, and may develop an identity around being a certain "type of
```

Comment:
again, i am not sure about the choice to swap quotes to italics, pair programming and talking stick I am very much on the fence (I might even think that those two should actually be links only, no additional formatting), and am happy to defer, but I think "talented" and "high ability" are in quotes to indicate that these are not values that we hold but are things that other people often say.  

at a minimum, I am confidence that "talented", "high ability" and "type of person" should be treated the same in this sentence so alternative to my suggestion is to swap `"` to `*` on type of person. 
```suggestion
are often praised as "talented" or having "high ability", and may develop an identity around being a certain "type of
```